src/components/ActivityButtons.tsx

```diff
@@ -66,6 +67,33 @@ function ClassManualOption(props: {
   );
 }
 
+function OverrideLocations(props: { state: State; secs: Sections }) {
+  const [isOverriding, setIsOverriding] = useState(false);
+  const { state, secs } = props;
+  const [room, setRoom] = useState(secs.roomOverride);
+  const onRelocate = () => {
+    setIsOverriding(true);
+  };
+  const onConfirm = () => {
+    secs.roomOverride = room.trim();
+    setIsOverriding(false);
+    state.updateActivities();
+  };
+  const onCancel = () => {
+    setIsOverriding(false);
+  };
+  return isOverriding
+    ? <Flex gap={1} mr = {1}><Input
+        value={room}
+        onChange={e => setRoom(e.target.value)}
+        fontWeight="bold"
+        placeholder="Somewhere" />
```

Comment:
I think something like "e.g., 26-100" would be a more helpful placeholder. A similar change could also be made to the location editor for custom activities, which currently is just the word "room". (Maybe it could be "e.g., W20-557"? [;) )](https://xkcd.com/541/)